benchmarks/benchmarks/bench_core.py

```diff
@@ -75,6 +76,15 @@ def time_triu_l10x10(self):
     def time_tril_l10x10(self):
         np.tril(self.l10x10)
 
+    def time_sort(self):
+        np.sort(self.arr)
```

Comment:
It might be nice to put the default `sort` benchmark in first (perhaps along with existing supported sorts like quicksort). That way we have a baseline in the live updating benchmarks. That way when this PR lands, people can see how much changing to radix benefits them.

ref: https://pv.github.io/numpy-bench/

Reply:
Done.

Reply:
Thanks for doing that. Have pulled out the other benchmarks into PR ( https://github.com/numpy/numpy/pull/12723 ) to help set a baseline for when this PR lands. Hope that is ok.